layouts/header.html

```diff
@@ -40,6 +40,7 @@
 
     <!-- Custom styles for this template -->
     <link href="/css/docs.css" rel="stylesheet">
+    <link href="/css/routing-tree-editor.css" rel="stylesheet">
```

Comment:
Yeah, the total file custom css size is 2kb, so the concern is if these are overwriting any other styles. I'm pretty sure they aren't interfering with any other styles, though.